.gitlab-ci.yml

```diff
@@ -0,0 +1,21 @@
+image: ubuntu:bionic
+
+variables:
+  GRADLE_VERSION: "5.4.1"
```

Comment:
I think for now we should stay with the same Gradle version as in .travis.yml: 4.10.3

Reply:
Sure, I'll change it. (At some point we might want to do builds with multiple Gradle versions as we are doing on Travis)

Reply:
Yes, is there something like the `matrix` availble?

Reply:
There is, so I can try to add that in a subsequent PR. I just wanted to get a basic build working first.